src/engine/strat_engine/backstore/backstore.rs

```diff
@@ -557,14 +556,19 @@ impl Backstore {
         &mut self,
         uuid: DevUuid,
         user_info: Option<&str>,
-    ) -> RenameAction<DevUuid> {
+    ) -> StratisResult<Option<DevUuid>> {
```

Comment:
Please update the header comment to describe the extended specification, i.e., the meaning of ```None``` and ```Some``` and extend the error message to the maximum available info, something like "No block device corresponding to device UUID ```<such and such>``` found". Someday, we may want to not ignore the error.

Reply:
Resolved.

Reply:
Resolved.